com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -583,10 +584,11 @@ internal void SetCachedParent(Transform parentTransform)
 
         internal (bool IsReparented, ulong? LatestParent) GetNetworkParenting() => (m_IsReparented, m_LatestParent);
 
-        internal void SetNetworkParenting(bool isReparented, ulong? latestParent)
+        internal void SetNetworkParenting(bool isReparented, ulong? latestParent, bool childWorldPositionStays)
```

Comment:
Is `childWorldPositionStays` the right name ?
I’d have expected `myWorldPositionStays` or `worldPositionStays`.

Reply:
Agreed - let's not rename the already-existing Unity parameter name to a new name.